src/api/controllers/followController.js

```diff
@@ -0,0 +1,57 @@
+import models from '../models/index';
+import status from '../../helpers/constants/status.codes';
+import errorSender from '../../helpers/error.sender';
+/**
+ * @class
+ */
+export default class Follow {
```

Comment:
@truestbyheart I think the class and model name should be nouns. Perhaps `Following` might work here.